common.lic

```diff
@@ -155,6 +155,12 @@ module DRC
         put message
         timer = Time.now
         next
+      when /^You must be standing/, /^You should stand up first/
```

Comment:
These are other positions you could find yourself in that would prevent movement that need you to stand back up
```suggestion
      when /^You must be standing/, /^You should stand up first/, /^You can't do that while (sitting|kneeling|lying)/
```